src/storage/clickhouse/clickhouse.service.ts

```diff
@@ -40,12 +41,8 @@ import {
   ValidatorIdentifications,
   ValidatorsStatusStats,
 } from './clickhouse.types';
-import migration_000001_init from './migrations/migration_000001_init';
-import migration_000002_validators from './migrations/migration_000002_validators';
-import migration_000003_attestations from './migrations/migration_000003_attestations';
-import migration_000004_proposes from './migrations/migration_000004_proposes';
-import migration_000005_sync from './migrations/migration_000005_sync';
-import migration_000006_attestations from './migrations/migration_000006_attestations';
+import migration_000000_summary from './migrations/migration_000000_summary';
+import migration_000001_indexes from './migrations/migration_000001_indexes';
 
```

Comment:
It brakes the ability to run the software ab ovo.

Reply:
Why?

Reply:
1. You've replaced all the tables with the single one, right? If so, no break, it's my fault...
2. ... but it worth adding migration removing unused tables (if it is not)

Reply:
We want that user doesn't accidentally lose his data when starting new version and could transfer old data by himself, if necessary. It will be noted in release notes

Reply:
Keep it as is and do not forget to clean up storage at in-house deployment.